thinkhazard/static/less/report.less

```diff
@@ -229,3 +229,38 @@ body {
     margin-bottom: 4px;
   }
 }
+
+.style-pdf-overview-btn {
+  padding: 7px 10px;
+  margin-top: 15px;
+  border: 1px solid #d9d9d9;
+  font-size: 14px;
+
+  -webkit-border-radius: 5px;
+     -moz-border-radius: 5px;
+          border-radius: 5px;
```

Comment:
If I remember well, there's a mixin in bootstrap or less to do this more automatically.